src/clustering/administration/main/command_line.cc

```diff
@@ -1519,6 +1545,11 @@ options::help_section_t get_network_options(const bool join_required, std::vecto
                                              strprintf("%d", port_defaults::peer_port)));
     help.add("--cluster-port port", "port for receiving connections from other nodes");
 
+    options_out->push_back(options::option_t(options::names_t("--cluster-reconnect-timeout"),
+                                             options::OPTIONAL,
+                                             strprintf("%d", cluster_defaults::reconnect_timeout)));
+    help.add("--cluster-reconnect-timeout ms", "maximum number of secconds to attempt reconnecting to a node before giving up");
```

Comment:
This is still saying "ms" in `"--cluster-reconnect-timeout ms"`, should be "secs".
There also is a typo in the help message: `secconds` instead of `seconds`.